frontend/src/libs/components/chat-sidebar/chat-element/chat-element.tsx

```diff
@@ -0,0 +1,47 @@
+import { Icon } from '#libs/components/components.js';
+import { IconNameToIcon } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { useCallback } from '#libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  chat: Chat;
+};
+
+type Chat = {
+  id: number;
+  name: string;
+};
+
+// mocked data
+const selectedChat = {
+  id: 1,
+};
+
+const ChatElement: React.FC<Properties> = ({ chat }) => {
```

Comment:
Maybe ChatItem would be more suitable here

Reply:
I cherry picked these changes from cp-55 alongside with the Icon component I needed, Yevhenii has been working on it

Reply:
We need to stay with only one version, please check it to prevent duplication of code after merging

Reply:
Mister @v0ldemar01 I just got an idea: 
chat-element.tsx was cherrypicked from cp-55, it's by any means not integrated to my branch and not used anywhere. Can I just delete and it won't be causing conflicts when Yevhenii will merging his PR?